src/common/Pagination/index.js

```diff
@@ -6,26 +6,33 @@ export const Pagination = () => {
   const pageNumber = useSelector(selectPage);
   const totalPages = useSelector(selectTotalPages);
   const dispatch = useDispatch();
+  const firstPage = pageNumber === 1;
+  const lastPage = pageNumber === totalPages;
```

Comment:
good idea, maybe firstPage rename to isFirstPage (isLstPage)

Reply:
You're right, that would be better!